src/app/components/Text/index.styled.tsx

```diff
@@ -0,0 +1,390 @@
+import { css, Theme } from '@emotion/react';
+
+import pixelsToRem from '../../utilities/pixelsToRem';
+
+/* 
+   Screen sizes for GEL Typography
+   These namings are based on the GEL description. They are also known as group A, group B and group D
+   Link to relevant GEL docs: http://www.bbc.co.uk/gel/guidelines/typography#type-sizes
+*/
+
+const GROUP_B_MIN_WIDTH_BP = pixelsToRem(320);
+const GROUP_D_MIN_WIDTH_BP = pixelsToRem(600);
+
+const GROUP_B_MIN_WIDTH_MQ = `@media (min-width: ${GROUP_B_MIN_WIDTH_BP}rem)`;
+const GROUP_D_MIN_WIDTH_MQ = `@media (min-width: ${GROUP_D_MIN_WIDTH_BP}rem)`;
+
+export const getBaseStyles = ({ palette }: Theme) =>
+  css({
+    color: palette.GREY_10,
+  });
+
+export const getAtlasSize = ({ typography }: Theme) =>
```

Comment:
These functions all seem very similar - they could probably be shared, something like this:
```
export const getAtlasSize = getSize('atlas');
export const getElephantSize = getSize('elephant');
// etc

const getSize = size => ({ typography }: Theme =>
  css({
    fontSize: typography.script[size].groupA.fontSize,
   // etc  
  })
);
```

Reply:
ooh nice! Will do this!

Reply:
Thanks for that. [It saved on a lot of lines of code](https://github.com/bbc/simorgh/pull/10236/commits/71ea282707de4fba6bb4b92034d3e5624910e8de#diff-2037600299c9ce66224f9e8aafb02b95905030dab1830d2b3a648ebec8534c21)